bot/exts/moderation/infraction/_scheduler.py

```diff
@@ -226,7 +223,13 @@ async def apply_infraction(
                 log.error(f"Deletion of {infr_type} infraction #{id_} failed with error code {e.status}.")
             infr_message = ""
         else:
+            log.info(f"Applied {purge}{infr_type} infraction #{id_} to {user}.")
             infr_message = f" **{purge}{' '.join(infr_type.split('_'))}** to {user.mention}{expiry_msg}{end_msg}"
+            if infr_type not in ("ban", "kick"):  # if we haven't already tried to send the DM
```

Comment:
```suggestion
            if infr_type not in ("ban", "kick"):  # If we haven't already tried to send the DM
```

Reply:
From the looks of it we do generally have the first word capitalised so will commit this :+1: